dll/cpl/sysdm/lang/fr-FR.rc

```diff
@@ -367,7 +367,7 @@ BEGIN
     IDS_HWPROFILE_ALREADY_IN_USE "Le nom de profil est déjà utilisé."
     IDS_HWPROFILE_PROFILE "Profil"
     IDS_HWPROFILE_WARNING "Attention"
-    IDS_ENVIRONMENT_WARNING "Any changes that have been done will be discarded and the variable's value will be edited as text."
-    IDS_ENVIRONMENT_WARNING_TITLE "System Properties"
-    IDS_FILE_BROWSE_FILTER "All Files (*.*)\0*.*\0"
+    IDS_ENVIRONMENT_WARNING "Tous les changements effectués seront abandonnés et la valeur de la variable sera éditée en texte."
```

Comment:
"éditée en texte" , or: "éditée comme texte" ?

Reply:
both looks good... what would you prefer ?

Reply:
let's see what @zefklop and @hpoussin think about it :thinking: